### ISSUE TITLE
Vote logging uses a UTC timestamp even though the server has a different configuration

### ISSUE THREAD

**Issue by catileptic** [code4romania/votong#209]
The VotONG server is configured to display timestamps using the `Europe/Bucharest` timezone ( [here](https://github.com/code4romania/votong/blob/develop/civil_society_vote/civil_society_vote/settings/base.py#L146) ). However, when we send logging e-mails to the logging inbox ( [here](https://github.com/code4romania/votong/blob/199421f1d6cdad97959ed1383780827926c710f5/civil_society_vote/hub/management/commands/init.py#L33) ), the timestamp in these emails it UTC. This timestamp should be the same one that we configure in the settings file.

**Comment by aramboi** [comment]
It should require little changes here: https://github.com/code4romania/votong/blob/7de95ba1c72f21913e44d79a45946637a16ba683/civil_society_vote/hub/views.py#L419

Something along the lines
```python
from django.utils import timezone
....
"timestamp": timezone.localtime(vote.created).isoformat(),
...
```

**Comment by aramboi** [comment]
fixed in #256
